categories.tex

```diff
@@ -17,7 +17,7 @@ \chapter{Category theory}
 If we add the requirement that the type $A_0$ of objects is a set, then we end up with a definition that behaves much like the traditional set-theoretic one.
 Following Toby Bartels, we call this notion a \emph{strict category}.
 \index{strict!category}%
-But we can also require a generalized version of the univalence axiom, identifying $(x=_{A_0} y)$ with the type $\mathsf{iso}(x,y)$ of isomorphisms from $x$ to $y$.
+But we can also instead require a generalized version of the univalence axiom, identifying $(x=_{A_0} y)$ with the type $\mathsf{iso}(x,y)$ of isomorphisms from $x$ to $y$.
```

Comment:
Without "instead" the text reads like we assume $A_0$ is a set *and* we also assume a generalized version of univalence. The word "instead" indicates an alternative. An alternative would be to say instead "Alternatively we can require ..."